http_logs/operations/default.json

```diff
@@ -2,7 +2,32 @@
       "name": "index-append",
       "operation-type": "bulk",
       "bulk-size": {{bulk_size | default(5000)}},
-      "ingest-percentage": {{ingest_percentage | default(100)}}
+      "ingest-percentage": {{ingest_percentage | default(100)}},
+      "corpora": "http_logs"
+    },
+    {
+      "name": "index-append-with-ingest-baseline-pipeline",
+      "operation-type": "bulk",
+      "bulk-size": {{bulk_size | default(5000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}},
+      "pipeline": "http-log-baseline-pipeline",
+      "corpora": "http_logs"
```

Comment:
it is using the `http_logs` ?

_assuming you meant `http_logs_unparsed` ...
It uses the pre-parsed version so not occur the over head of parsing, but still occur the overhead of indexing all of the fields. If I were to use the `http_logs_unparsed` it would end up indexing only the message field which is actually index:false.  The net difference between the baseline with pre-parsed vs. grok with un-parsed should be the overhead of grok plus the (un-avoidable) overhead of the index:false message field.

Reply:
Maybe I should have not stated the obvious... . What I wanted to convey here is that my initial thought was that you should use `http_logs_unparsed`  but then I realized that `http_logs` is indeed correct... . Sorry for the confusion. :)